lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -3676,7 +3690,7 @@ public SecurityAdvice isAllowed(String userId, String function, String reference
     // the group list, so you need to do i.setGroups().
        public List<String> setItemGroups (SimplePageItem i, String[] groups) {
 	   // can't allow groups on student pages
-	   if (getCurrentPage().getOwner() != null)
+	   if (isStudentPage(getCurrentPage()))
 	       return null;
```

Comment:
While you're on this line, you could improve this single line conditional to include braces